python/highlevelil.py

```diff
@@ -1646,6 +1646,10 @@ def detailed_operands(self) -> List[Tuple[str, HighLevelILOperandType, str]]:
 			("constant", self.constant, "int"),
 		]
 
+	@property
+	def string(self) -> Optional[binaryview.StringReference]:
```

Comment:
I rebased this branch on `dev` and I have no Python Console.

After some debugging (and setting up a completely new, fresh Python just for a debug build), doing an `import binaryninja` with the built API from this branch results in a big traceback because:

```
    class HighLevelILConstPtr(HighLevelILInstruction, Constant):
  File "/Users/fuzyll/Builds/binja-dbg/out/binaryninja.app/Contents/Resources/python/binaryninja/highlevelil.py", line 1650, in HighLevelILConstPtr
    def string(self) -> Optional[binaryview.StringReference]:
AttributeError: partially initialized module 'binaryninja.binaryview' has no attribute 'StringReference' (most likely due to a circular import)
```

Does this actually work? Or, am I Doing It Wrong..?